api/app/jobs/common_model_fetchers.py

```diff
@@ -245,41 +246,40 @@ def transform(long, lat):
         # 2020 Dec 15, Sybrand: Encountered situation where tmp_tgl_2 was None, add this workaround for it.
         # NOTE: Not sure why this value would ever be None. This could happen if for whatever reason, the
         # tmp_tgl_2 layer failed to download and process, while other layers did.
-        if prediction.tmp_tgl_2 is None:
-            logger.warning('tmp_tgl_2 is None for ModelRunGridSubsetPrediction.id == %s', prediction.id)
+        if prediction.tmp_tgl_2 is None or len(prediction.tmp_tgl_2) == 0:
+            logger.warning('tmp_tgl_2 is None or empty for ModelRunGridSubsetPrediction.id == %s', prediction.id)
         else:
-            station_prediction.tmp_tgl_2 = griddata(
-                points, prediction.tmp_tgl_2, coordinate, method='linear')[0]
+            station_prediction.tmp_tgl_2 = prediction.tmp_tgl_2[0]
```

Comment:
Very minor, not code related. The whole docstring/note for this method explains the use of griddata still, starts on line 204. Not sure if you're ready to remove that

Reply:
Thanks! I'll update the docstring.